static/index.js

```diff
@@ -1,13 +1,36 @@
 window.onload = () => {
-  const item1 = document.createElement("li");
-  const item2 = document.createElement("li");
-  const item3 = document.createElement("li");
+  // const item1 = document.createElement("li");
+  // const item2 = document.createElement("li");
+  // const item3 = document.createElement("li");
 
-  item1.innerText = "take garbage out";
-  item2.innerText = "Play video game";
-  item3.innerText = "Take over the world";
+  // item1.innerText = "take garbage out";
+  // item2.innerText = "Play video game";
+  // item3.innerText = "Take over the world";
 
-  const listItems = document.createElement("ul");
-  listItems.append(item1, item2, item3);
-  const vanillaList = document.getElementById("list").appendChild(listItems);
+  // listItems.append(item1, item2, item3);
+  // const vanillaList = document.getElementById("list").appendChild(listItems);
+
+  const items = [
+    { title: "take garbage out", checked: true },
+    { title: "Play video games", checked: false },
+    { title: "Take over the world", checked: true }
+  ];
+  const todoItem = items => {
```

Comment:
todoItem should take in a single item, your variable doesn't represent that here